schema/spacewalk/common/data/rhnServerServerGroupArchCompat.sql

```diff
@@ -1261,4 +1261,32 @@ insert into rhnServerServerGroupArchCompat ( server_arch_id, server_group_type )
 	values (lookup_server_arch('arm64-debian-linux'),
             lookup_sg_type('proxy_entitled'));
             
+insert into rhnServerServerGroupArchCompat ( server_arch_id, server_group_type )
+	values (lookup_server_arch('armhf-debian-linux'),
+            lookup_sg_type('enterprise_entitled'));
+
+insert into rhnServerServerGroupArchCompat ( server_arch_id, server_group_type )
+	values (lookup_server_arch('armhf-debian-linux'),
+            lookup_sg_type('virtualization_host'));
+
+insert into rhnServerServerGroupArchCompat ( server_arch_id, server_group_type )
+	values (lookup_server_arch('armhf-debian-linux'),
+            lookup_sg_type('bootstrap_entitled'));
+
+insert into rhnServerServerGroupArchCompat ( server_arch_id, server_group_type )
+	values (lookup_server_arch('armhf-debian-linux'),
+            lookup_sg_type('salt_entitled'));
+
+insert into rhnServerServerGroupArchCompat ( server_arch_id, server_group_type )
+	values (lookup_server_arch('armhf-debian-linux'),
+            lookup_sg_type('foreign_entitled'));
+
+insert into rhnServerServerGroupArchCompat ( server_arch_id, server_group_type )
+	values (lookup_server_arch('armhf-debian-linux'),
+            lookup_sg_type('container_build_host'));
+
+insert into rhnServerServerGroupArchCompat ( server_arch_id, server_group_type )
+	values (lookup_server_arch('armhf-debian-linux'),
+            lookup_sg_type('monitoring_entitled'));
+
```

Comment:
Shouldn't we add also `proxy_entitled` for `armhf-debian-linux`? I see we have it `arm64-debian-linux` and other ARM related archs.

Reply:
Doesn't it mean Uyuni Proxy based on armhf debian client? I don't think it's possible.